core/src/main/java/org/apache/iceberg/DataFilesTable.java

```diff
@@ -120,7 +127,8 @@ protected long targetSplitSize(TableOperations ops) {
       super(DataFiles.fromManifest(manifest), null, schemaString, specString, residuals);
       this.io = io;
       this.manifest = manifest;
-      this.schema = schema;
+      //Ignore SpecID when reading
```

Comment:
Is this comment allowed by the linter? I thought it wasn't allowed to omit the space after `//`.

Reply:
Sorry this is WIP I was mostly getting this out as a "Does this idea even make sense" sort of thing. I wanted to demonstrate in code what we were thinking